### ISSUE TITLE
Server Vetur Language Server failed to start: Request initialize failed with message: The "path" argument must be of type string. Received type undefined

### ISSUE THREAD

**Issue by mklingn** [neoclide/coc-vetur#15]
First off: Thanks for the great tool.

I found an issue  with release version 1.1.4. Below is the output of `:CocInfo`.

---

## versions                                                                                                                                                                  
                                                                                                                                                                             
vim version: NVIM v0.3.4                                                                                                                                                     
node version: v12.7.0                                                                                                                                                        
coc.nvim version: 0.0.74-c97d41aa1e                                                                                                                                          
term: screen                                                                                                                                                                 
platform: linux                                                                                                                                                              
                                                                                                                                                                             
## Messages                                                                                                                                                                  
[coc.nvim] Request initialize failed with message: The "path" argument must be of type string. Received type undefined                                                       
[coc.nvim] Server Vetur Language Server failed to start: Request initialize failed with message: The "path" argument must be of type string. Received type undefined         

---

Downgrading to release version 1.1.3 fixes the issue.

**Comment by kqito** [comment]
I have the same issue.

**Comment by donsuhr** [comment]
Some additional messages:

```
[coc.nvim] Updated extension: coc-vetur to 1.1.4
[coc.nvim] Update completed
[coc.nvim] error: UnhandledRejection: Request shutdown failed with message: Cannot read property 'languageServiceRefCount' of undefined
[coc.nvim] error: Error: Request shutdown failed with message: Cannot read property 'languageServiceRefCount' of undefined
[coc.nvim] error:     at handleResponse (/Users/bill/.vim/plugged/coc.nvim/build/index.js:38151:48)
[coc.nvim] error:     at processMessageQueue (/Users/bill/.vim/plugged/coc.nvim/build/index.js:37978:17)
[coc.nvim] error:     at Immediate.setImmediate (/Users/bill/.vim/plugged/coc.nvim/build/index.js:37962:13)
[coc.nvim] error:     at runCallback (timers.js:705:18)
[coc.nvim] error:     at tryOnImmediate (timers.js:676:5)
[coc.nvim] error:     at processImmediate (timers.js:658:5)
[coc.nvim] error: UnhandledRejection: Request shutdown failed with message: Cannot read property 'languageServiceRefCount' of undefined
[coc.nvim] error: Error: Request shutdown failed with message: Cannot read property 'languageServiceRefCount' of undefined
[coc.nvim] error:     at handleResponse (/Users/bill/.vim/plugged/coc.nvim/build/index.js:38151:48)
[coc.nvim] error:     at processMessageQueue (/Users/bill/.vim/plugged/coc.nvim/build/index.js:37978:17)
[coc.nvim] error:     at Immediate.setImmediate (/Users/bill/.vim/plugged/coc.nvim/build/index.js:37962:13)
[coc.nvim] error:     at runCallback (timers.js:705:18)
[coc.nvim] error:     at tryOnImmediate (timers.js:676:5)
[coc.nvim] error:     at processImmediate (timers.js:658:5)
[coc.nvim] Request initialize failed with message: The "path" argument must be of type string. Received type undefined
[coc.nvim] Server Vetur Language Server failed to start: Request initialize failed with message: The "path" argument must be of type string. Received type undefined
```

**Comment by ladevieq** [comment]
You can also change package.json and stay in 1.1.4 :

```JSON
-    "coc.nvim": "^0.0.73",
+    "coc.nvim": "0.0.71",
-    "vscode-languageserver-protocol": "^3.15.0-next.6",
+    "vscode-languageserver-protocol": "3.15.0-next.6",
-    "vue-language-server": "^0.0.60"
+    "vue-language-server": "0.0.57"
```

**Comment by mklingn** [comment]
For anybody wondering, the diffs from @ladevieq are meant to be applied to *~/.local/share/nvim/plugged/coc.nvim/package.json*. 

One could probably also fix coc-vetur to 1.1.3 in *~/.config/coc/extensions/package.json*. 

But most likely one will forget having done so and experience strange effects in the not so far future...
